Mathlib/SetTheory/Ordinal/Exponential.lean

```diff
@@ -21,46 +21,58 @@ universe u v w
 
 namespace Ordinal
 
-/-- The ordinal exponential, defined by transfinite recursion. -/
-instance pow : Pow Ordinal Ordinal :=
-  ⟨fun a b => if a = 0 then 1 - b else limitRecOn b 1 (fun _ IH => IH * a) fun b _ => bsup.{u, u} b⟩
+/-- The ordinal exponential, defined by transfinite recursion.
 
-theorem opow_def (a b : Ordinal) :
-    a ^ b = if a = 0 then 1 - b else limitRecOn b 1 (fun _ IH => IH * a) fun b _ => bsup.{u, u} b :=
-  rfl
+We call this `opow` in theorems in order to disambiguiate from other exponentials. -/
```

Comment:
This comment is a useful one - might be nice to also add it to the module docstring.